openreview/conference/helpers.py

```diff
@@ -138,7 +139,7 @@ def get_conference_builder(client, request_form_id, support_user='OpenReview.net
     create_review_invitation = create_groups and note.content.get('make_reviews_public', None)
 
     author_names_revealed = 'Reveal author identities of all submissions to the public' in note.content.get('reveal_authors', '') or 'Reveal author identities of only accepted submissions to the public' in note.content.get('reveal_authors', '')
-    papers_released = 'Release all submissions to the public'in note.content.get('release_submissions', '') or 'Release only accepted submission to the public' in note.content.get('release_submissions', '')
+    papers_released = 'Release all submissions to the public'in note.content.get('release_submissions', '') or 'Release only accepted submission to the public' in note.content.get('release_submissions', '') or 'Everyone for accepted submissions and only assigned program committee for rejected submissions' in note.content.get('submission_readers', '')
```

Comment:
@melisabok how should we define `papers_released` now? I think the only place we need this for is to not override `venueid` and `venue` for blind submissions, so perhaps we can just check in `create_blind_submissions()` if these fields exist already and add them to content? Since we are using `submission_readers` everywhere now we can't really distinguish if `post_decision` has been called or not

used here: https://github.com/openreview/openreview-py/blob/c8854ef6cf6a8892322b5370d20d690d8bd5fe6a/openreview/conference/builder.py#L887

Reply:
That's a good idea. Let's remove `papers_released` and keep the values `venue`  and `venueid` if they already exist, I think `_bibtex` too?